src/SystemViewToolbar.tsx

```diff
@@ -0,0 +1,209 @@
+
+  const { zoomIn, zoomOut, fitView } = useZoomPanHelper();
+  const setInteractive = useStoreActions((actions) => actions.setInteractive);
+  const isInteractive = useStoreState((s) => s.nodesDraggable && s.nodesConnectable && s.elementsSelectable);
+
+  const onToggleOrientation = useCallback(() => {
+    setLROrientation(!lrOrientation);
+    props.onToggleOrientation?.(lrOrientation);
+  }, [lrOrientation, props.onToggleOrientation]);
+
+  const onZoomInHandler = useCallback(() => {
+    zoomIn?.();
+    props.onZoomIn?.();
+  }, [zoomIn, props.onZoomIn]);
+
+  const onZoomOutHandler = useCallback(() => {
+    zoomOut?.();
+    props.onZoomOut?.();
+  }, [zoomOut, props.onZoomOut]);
+
+  const onFitViewHandler = useCallback(() => {
+    fitView?.(props.fitViewParams);
+    props.onFitview?.();
+  }, [fitView, props.fitViewParams, props.onFitview]);
+
+  const onInteractiveChangeHandler = useCallback(() => {
+    setInteractive?.(!isInteractive);
+    props.onInteractiveChange?.(!isInteractive);
+  }, [isInteractive, setInteractive, props.onInteractiveChange]);
+
+  return (
+    <Toolbar>
+      <br />
+      <ExpandingToolbar
+        tooltip="Group nodes"
+        iconName={
+          <FoxgloveIcon style={{ color: "white" }}>
+            <GroupIcon />
+          </FoxgloveIcon>
+        }
+        // className={styles.button}
+        selectedTab={selectedTab}
+        onSelectTab={(newSelectedTab) => {
+          setSelectedTab(newSelectedTab!)
+        }}
+      >
+        <ToolGroup name={"Node Grouping"}>
+          <>
+            <br />
+            <SegmentedControl
+              options={optionArray}
+              selectedId={selectedId}
+              onChange={(newId) => setSelectedId(newId)}
+            />
+          </>
+        </ToolGroup>
+      </ExpandingToolbar>
+      <ExpandingToolbar
+        tooltip="Select nodes to display"
+        iconName={
+          <FoxgloveIcon style={{ color: "white" }}>
+            <SelectionIcon />
+          </FoxgloveIcon>
+        }
+        selectedTab={selectedTab2}
+        onSelectTab={(newSelectedTab) => {
+          setSelectedTab2(newSelectedTab!)
+        }}
+      >
+        <ToolGroup name={"Node List"}>
+          <NodeList nodes={props.nodes} edges={props.edges} />
+        </ToolGroup>
+        <ToolGroup name={"Options"}>
+          <>
+            <Checkbox
+              label="Automatically display new nodes"
+              checked={false}
+              onChange={() => console.log("onChange")}
+            />
+            <Checkbox
+              label="Include hidden nodes"
+              checked={false}
+              onChange={() => console.log("onChange")}
+            />
+            <Checkbox
+              label="Include hidden topics"
+              checked={false}
+              onChange={() => console.log("onChange")}
+            />
+          </>
+        </ToolGroup>
+      </ExpandingToolbar>
+      <div className={styles.button}>
```

Comment:
These were all `styles.buttons`, which didn't seem to exist in the imported type. I've changed them to `styles.button`, which is on the type. I'm not sure if this is what was intended.